continuous-integration/requirements.Rmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "Requirements for continuous integration"
+maintainer: "Thierry Onkelinx"
+output: html_document
+---
+
+# Must have
+
+- start automatically when pushing a git repository on Github or Bitbucket
+- start automatically when creating a pull request on Github or Bitbucket
+- able to use a Docker image to setup the environment in which CI runs
```

Comment:
Hence, CI does not need to be Docker-based, but in order to comply to current projects, the support for Docker is required.